admin_manual/configuration/server/oc_server_tuning.rst

```diff
@@ -24,8 +24,74 @@ Memory cache configuration for the ownCloud server is no longer automatic in
 ownCloud 8.1 and up, but must be installed and configured. See      
 :doc:`caching_configuration`.
 
-Using MariaDB/MySQL instead of SQLite
--------------------------------------
+Tune Redis
+~~~~~~~~~~
+
+Here is a brief guide for tuning Redis to improve the performance of your ownCloud installation, when working with large instances.
+
+Using Redis-based Transactional File Locking
```

Comment:
I would switch this section with the "Tune redis" header, e.g.:

1. Caching -> Headline for Caching hints (memcache and redis)
1.1 Use / configure memory cache
1.2 Use redis for transactional file locking
1.1.1 Tuning redis -> People might be able to use redis as it is available by the hoster but not able to do the other performance hints due to missing permissions

Reply:
I see what you're saying, @kdslkdsaldsal, and thanks for sharing further input. However, the recommendations are not mandatory. Users are free to use them if & when they are able to do so. Given that, I'm fine with letting it stay as it is.

Reply:
@settermjd I still think the headlines are wrong / misleading for the simple reason that ``Using Redis-based Transactional File Locking`` is now / after this PR a sub-headline of ``Redis Tuning``.

However ``Using Redis-based Transactional File Locking`` is not a ``Redis Tuning`` but an ``ownCloud Server`` tuning.

Maybe a little bit different to my suggestion above to make the differences more clear:

Enable Memory Caching
-------------------------------
-> You can use Redis, APCu or Memcached for Memory Caching

Using Redis-based Transactional File Locking
----------------------------------------------------------
-> Only Redis is available for Transactional File locking

Redis Tuning
-----------------

-> Redis tuning helps for both, File Locking (if used) and/or Memory Caching (if Redis is used)

TCP-Backlog
^^^^^^^^^^^

The rest of the Redit stuff.

Database Tuning
---------------

Reply:
Gotcha! Will make the relevant changes.